Python/thread_pthread.h

```diff
@@ -143,6 +143,7 @@ typedef struct {
 } pthread_lock;
 
 #define CHECK_STATUS(name)  if (status != 0) { perror(name); error = 1; }
+#define CHECK_STATUS_PTHREAD(name)  if (status != 0) { char* strerror_msg = strerror(status);  fprintf(stderr, "%s: %s\n", name, strerror_msg); }
```

Comment:
temporary variable seems redundant.
`fprintf(stderr, "%s: %s\n", name, strerror(status))` is enough, maybe.

You removed `error = 1;`.  Is it intended change?

Please wrap line within 79 columns.